bzt/modules/provisioning.py

```diff
@@ -75,12 +75,12 @@ def prepare(self):
     def startup(self):
         self.start_time = time.time()
 
-        self.available_slots = self.settings.get("capacity", None)
-        if not self.available_slots:
-            if self.settings.get("sequential", False):
-                self.available_slots = 1
-            else:
-                self.available_slots = sys.maxsize      # no limit
+        if self.settings.get("sequential", False):
+            self.available_slots = 1
+        else:
+            self.available_slots = self.settings.get("capacity", None)
+            if not self.available_slots:
+                self.available_slots = sys.maxsize  # no limit
```

Comment:
The bug is not about the priority , But we use 2 config items CAPACITY and SEQUENTIAL to control 1 action: thoughput.
If 'sequential' is given to 'False' as default in configfile and then I use 'capacity' to 10 in my test.yml , the 'capacity' 's priority should be higher than default 'sequential'. So, I suggest we use only one 'capacity', it is enough. '1' is sequential, more than 1 is capacity, None is sys.maxsize.

Reply:
Hello Tao,

I understand your point. And it is completely logical, why we should have two parameters, when we can have just one. But it is not that simple though. Yes, 'capacity' kind of contains 'sequential', but we focus on simplicity and on users, who probably would like just to use 'sequential' and not dig deep into 'capacity'. That's actually why we have this simple '-sequential' command-line option and not '-capacity=X'.

BR,
Alla

Reply:
OK , That's reasonable.